Aptoide/aptoide-client-v8
# Feature/asv 1245 home dialog for promotions

**What does this PR do?**

This PR has 3 aims:
- Adding the icon to the home action bar when promotions are available. If there are available promotions, the icon shows a ticker on top of it showing how many of them you can claim;
-  Adding the home dialog on first launch of the app with promotions available;
- Implementing the request to the get promotions API;

**Database changed?**

No

**Where should the reviewer start?**


**How should this be manually tested?**

Check out the flows described above and check if everything matches what's coming in charles and works correctly;

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1245](<https://aptoide.atlassian.net/browse/ASV-1245>)

**Code Review Checklist**
